src/Message/ButtonReplyMessage.php

```diff
@@ -8,29 +8,21 @@ class ButtonReplyMessage extends Message
 {
     protected string $type = 'button';
 
-    private ?string $header;
-
     private string $body;
 
     private ?string $footer;
 
     private ButtonAction $action;
 
-    public function __construct(string $to, string $body, ButtonAction $action, ?string $header = null, ?string $footer = null, ?string $reply_to = null)
+    public function __construct(string $to, string $body, ButtonAction $action, ?string $footer = null, ?string $reply_to = null)
```

Comment:
ButtonReplyMessage is not a final class, that is, someone else may have extended this class in their project.

If you change the order or remove parameters from the constructor you are breaking all projects that update the version of this package.

Reply:
Reverted and not included in unbundled PRs